util/src/main/resources/SummaryReport.ftl

```diff
@@ -82,7 +88,7 @@
     <#if svSummaryMap?has_content>
       <div>
         <h2>StatVarObservations by StatVar</h2>
-        <table width="95%">
+        <table id="statvars-table" width="95%">
```

Comment:
since the new DataTable has borders, maybe we can get rid of the borders for this table in the css

Reply:
done. [code sandbox](https://codesandbox.io/s/green-smoke-ghjtv5?file=/index.html) updated with the new HTML.

note that there are minor changes in the styling of DataTables that are 1) negligible, 2) requires non-trivial to match the rest of the document. I'm opting to not waste effort on this.